portal/src/texts/universell-utforming/Roller.mdx

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
Hvis dette er noe som skal inn, så trenger jeg hjelp til å lande på et språk og formuleringer som vi som organisasjon kan stå bak. Per nå skinner nok min mening litt hard igjennom.

Reply:
Justerte på rollene til å være med områder, hvor teamene har ansvar for å dekke områdene, ikke at hver enkelt person i området må alene dekke alle forventningene.